lib/pages/profile_info/profile_info.dart

```diff
@@ -27,8 +29,9 @@ class ProfileInfoController extends State<ProfileInfo> {
       ? Matrix.of(context).client.getRoomById(widget.roomId!)
       : null;
 
-  User? get user =>
-      room?.unsafeGetUserFromMemoryOrFallback(room?.directChatMatrixID ?? '');
+  User? get user => room?.unsafeGetUserFromMemoryOrFallback(
+        room?.directChatMatrixID ?? widget.userId ?? '',
```

Comment:
Can you explain why it's not `widget.userId` first ?

Reply:
no worries, room or userId cannot both not null, so the priority is same